HomeWork/src/main/java/io/FileLoggerConfigurationLoader.java

```diff
@@ -0,0 +1,28 @@
+package io;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.Properties;
+
+public class FileLoggerConfigurationLoader {
+
+    FileLoggerConfiguration configuration;
```

Comment:
Remove this field. This class should not have any fields. `FileLoggerConfiguration` should be only returned in the method `load`.